examples/CMakeLists.txt

```diff
@@ -9,13 +9,16 @@ set(CALLER_SOURCES caller.cpp)
 set(CALLEE_SOURCES callee.cpp)
 set(PUBLISHER_SOURCES publisher.cpp)
 set(SUBSCRIBER_SOURCES subscriber.cpp)
+set(SUBSCRIBER_WAMPCRA_SOURCES subscriber_wampcra.cpp)
 
 add_executable(caller ${CALLER_SOURCES} ${PUBLIC_HEADERS})
 add_executable(callee ${CALLEE_SOURCES} ${PUBLIC_HEADERS})
 add_executable(publisher ${PUBLISHER_SOURCES} ${PUBLIC_HEADERS})
 add_executable(subscriber ${SUBSCRIBER_SOURCES} ${PUBLIC_HEADERS})
+add_executable(subscriber_wampcra ${SUBSCRIBER_WAMPCRA_SOURCES} ${PUBLIC_HEADERS})
 
 target_link_libraries(caller examples_parameters)
 target_link_libraries(callee examples_parameters)
 target_link_libraries(publisher examples_parameters)
 target_link_libraries(subscriber examples_parameters)
+target_link_libraries(subscriber_wampcra examples_parameters crypto)
```

Comment:
Not sure how to check for the existence of a library (little rusty with Makefiles)